utils/docker/DOCKER_README.md

```diff
@@ -0,0 +1,124 @@
+## Setup
```

Comment:
We should add somewhere that this has only been tested on machines with NVIDIA graphics cards.
Maybe add here as a note/warning/... ?

Alternatively we could add instructions for AMD, but this might be a lot of work, and probably hard to test :sweat_smile:

Reply:
It's very unclear if Gazebo even uses the GPU. I read somewhere there's some plugins (like for LIDAR simulation) that use it, but normally not. Then again on the install page it says you need a dedicated GPU, and recommends nvidia for Ubuntu (http://gazebosim.org/tutorials?tut=guided_b1&cat=)